piprot/piprot.py

```diff
@@ -263,7 +263,7 @@ def main(
     session = FuturesSession()
     results = []
 
-    for req, version, ignore in requirements:
+    for req, version, ignore in list(set(requirements)):
```

Comment:
@sesh Haha, I knew this was too simple a solution -- I overlooked the verbatim flag. Okay, I'll make some changes soon and ping you when I roll out an update.